modules/dhcp_native_ms/dhcp_native_ms_main.rb

```diff
@@ -25,7 +25,7 @@ def del_record(record)
     def add_record(options)
       name, ip_address, mac_address, subnet_address, options = clean_up_add_record_parameters(options)
 
-      validate_ip(ip_address)
+      validate_ip(ip_address, 4)
```

Comment:
Yes, this is correct - do this for libvirt and isc, but not for the common module.